evenorodd.py

```diff
@@ -1,4 +1,8 @@
 
-
+# returns the word even if the provided number is even, or odd if it's odd
 def even_or_odd(number):
-    return "blah"
+    result = number%2
+    if result == 0:
+        return "even"
+    elif result != 0:
```

Comment:
I think it would be a smidge easier to thing through if this was just 'else' rather than adding in another if to contend with. Obviously the result in the code is the same :-)

Reply:
I don't remember the syntax for that, so although I thought it would be the simplest option, I decided to do it that way :-)